test/distributed/optim/test_zero_redundancy_optimizer.py

```diff
@@ -224,24 +228,46 @@ def test_step(self):
             m.bias.data = torch.tensor([2.0])
             m.to(self.device)
 
-            o = ZeroRedundancyOptimizer(m.parameters(), optimizer_class=SGD, lr=0.1)
+            lr = 0.1
+            o = ZeroRedundancyOptimizer(m.parameters(), optimizer_class=SGD, lr=lr)
             y = m(x)
             y.backward(x)
             for p in m.parameters():
                 dist.all_reduce(p.grad.data, op=dist.ReduceOp.SUM)
                 p.grad.data /= self.world_size
             o.step()
-            self.assertEqual(m.weight, torch.tensor([[0.75]], device=self.device))
-            self.assertEqual(m.bias, torch.tensor([1.85], device=self.device))
+
+            # for y = wx + b, dy / dw = x and dy / db = 1
```

Comment:
Not sure how you guys feel about leaving this verbose comment in the test file. I included it because the math for the expected `m.weight` and `m.bias` values are non-obvious and did not want to hard-code them.